app/src/main/java/org/fossasia/openevent/general/speakers/Speaker.kt

```diff
@@ -18,9 +19,9 @@ data class Speaker(
     val name: String?,
     val email: String?,
     val photoUrl: String?,
-    val shortBiography: String?,
-    val longBiography: String?,
-    val speakingExperience: String?,
+    var shortBiography: String?,
+    var longBiography: String?,
+    var speakingExperience: String?,
```

Comment:
See how it is done for event description

Reply:
val description = event.description.nullToEmpty().stripHtml()
        if (!description.isNullOrEmpty()) {
            setTextField(rootView.eventDescription, description)

Reply:
It's done like this for event description. Should I do it like this?

Reply:
Yes

Reply:
done